whois-client/src/main/java/net/ripe/db/whois/common/sso/SsoTranslation.java

```diff
@@ -0,0 +1,52 @@
+package net.ripe.db.whois.common.sso;
+
+
+import com.google.common.collect.Maps;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+
+public class SsoTranslation {
```

Comment:
We already have a class for called SsoTranslator what is difference between this class? THis name is quite confusing with the other similar name

Reply:
yes, this code was a bit tricky I had to do a draft in a paper to understand